web/quickfeed_service.go

```diff
@@ -364,8 +364,11 @@ func (s *QuickFeedService) GetSubmissions(ctx context.Context, in *connect.Reque
 	return connect.NewResponse(submissions), nil
 }
 
-// GetSubmissionsByCourse returns all the latest submissions
-// for every individual or group course assignment for all course students/groups.
+// GetSubmissionsByCourse returns a map of submissions for the given course ID.
+// The map is keyed by either the group ID or enrollment ID depending on request type.
```

Comment:
Comment on the general behavior. I wonder if we should instead add a separate `FetchMode` for `EnrollmentID`??